app/policies/company_policy.rb

```diff
@@ -18,14 +18,7 @@ def create?
   end
 
   def update?
-    user.admin? || is_in_company?
+    user.admin? || is_in_company?(record)
```

Comment:
Either ask the company if this user is in it (will need to write a method) or use the already existing method in the User class `is_in_company_numbered?(company_num)`

Reply:
See related comments.